homeassistant/components/aussie_broadband/config_flow.py

```diff
@@ -0,0 +1,192 @@
+        errors: dict[str, str] = {}
+        if user_input and user_input.get(CONF_USERNAME):
+            self._reauth_username = user_input[CONF_USERNAME]
+
+        elif user_input and user_input.get(CONF_PASSWORD):
+            data = {
+                CONF_USERNAME: self._reauth_username,
+                CONF_PASSWORD: user_input[CONF_PASSWORD],
+            }
+
+            errors = await self.auth(data)
+            if errors == {}:
+                entry = await self.async_set_unique_id(self._reauth_username)
+                if entry:
+                    self.hass.config_entries.async_update_entry(
+                        entry,
+                        data=data,
+                    )
+                    await self.hass.config_entries.async_reload(entry.entry_id)
+                    return self.async_abort(reason="reauth_successful")
+                return self.async_create_entry(title=self._reauth_username, data=data)
+
+        return self.async_show_form(
+            step_id="reauth",
+            description_placeholders={"username": self._reauth_username},
+            data_schema=vol.Schema(
+                {
+                    vol.Required(CONF_PASSWORD): str,
+                }
+            ),
+            errors=errors,
+        )
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: config_entries.ConfigEntry,
+    ) -> config_entries.OptionsFlow:
+        """Get the options flow for this handler."""
+        return OptionsFlowHandler(config_entry)
+
+
+class OptionsFlowHandler(config_entries.OptionsFlow):
+    """Options flow for picking services."""
+
+    def __init__(self, config_entry: config_entries.ConfigEntry) -> None:
+        """Initialize options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(self, user_input=None):
+        """Manage the options."""
+        if user_input is not None:
+            return self.async_create_entry(title="", data=user_input)
+
+        if self.config_entry.state != config_entries.ConfigEntryState.LOADED:
+            return self.async_abort(reason="unknown")
+        data = self.hass.data[DOMAIN][self.config_entry.entry_id]
+        try:
+            services = await data["client"].get_services()
+        except AuthenticationException:
+            return self.async_abort(reason="invalid_auth")
+        except ClientError:
+            return self.async_abort(reason="cannot_connect")
+        service_options = {str(s[SERVICE_ID]): s["description"] for s in services}
+        return self.async_show_form(
+            step_id="init",
+            data_schema=vol.Schema(
+                {
+                    vol.Required(
+                        CONF_SERVICES,
+                        default=self.config_entry.options.get(CONF_SERVICES),
+                    ): cv.multi_select(service_options),
+                    vol.Optional(
+                        CONF_SCAN_INTERVAL,
```

Comment:
This should not be an option.
There should be a sane default and if the user needs to adjust they should disable polling for the config entry and implement a custom schedule with automation.

<https://www.home-assistant.io/blog/2021/06/02/release-20216/#disable-polling-updates-on-any-integration>

Reply:
@ludeeus this has been removed.